cluster/gce/coreos/helper.sh

```diff
@@ -69,9 +69,6 @@ ENABLE_CLUSTER_DNS=$(yaml-quote ${ENABLE_CLUSTER_DNS:-false})
 DNS_REPLICAS=$(yaml-quote ${DNS_REPLICAS:-})
 DNS_SERVER_IP=$(yaml-quote ${DNS_SERVER_IP:-})
 DNS_DOMAIN=$(yaml-quote ${DNS_DOMAIN:-})
-KUBE_USER=$(yaml-quote ${KUBE_USER})
-KUBE_PASSWORD=$(yaml-quote ${KUBE_PASSWORD})
-KUBE_BEARER_TOKEN=$(yaml-quote ${KUBE_BEARER_TOKEN})
```

Comment:
I think we need to change this https://github.com/GoogleCloudPlatform/kubernetes/blob/2f2816368f667bb81e6f59f910b0353604a6ccda/cluster/gce/coreos/node.yaml#L20 to use KUBELET_TOKEN. testing.